src/debugAdapter/goDebug.ts

```diff
@@ -1586,9 +1587,18 @@ export class GoDebugSession extends LoggingDebugSession {
 				log('EvaluateResponse');
 			},
 			(err) => {
+				let dest: ErrorDestination;
+				// No need to repeatedly show the error pop-up when expressions
+				// are continiously reevaluated in the Watch panel, which
+				// already displays errors.
+				if (args.context === 'watch') {
+					dest = null;
+				} else {
+					dest = ErrorDestination.User;
+				}
 				this.sendErrorResponse(response, 2009, 'Unable to eval expression: "{e}"', {
 					e: err.toString()
-				});
+				}, dest);
```

Comment:
Is the default destination always user if nothing is passed here?

Reply:
Yes, the [default is to show to the user](https://github.com/microsoft/vscode-debugadapter-node/blob/f0bef573e171c558ffdeba3c3a5c74f57378dd49/adapter/src/debugSession.ts#L432), but I decided to spell out both options explicitly in the if-else, so it is clear what is going on.

Reply:
cool, thanks!